apps/customer/src/app/(tabs)/main/_components/MainStoreCard.tsx

```
@@ -25,37 +23,80 @@ export default function MainStoreCard({ item }: MainStoreCardProps) {
-              </span>
-            </div>
           </div>
         </div>
       </Card>
     </button>
   );
+}
+
+function mapServerTagToLabel(tag: StoreTag) {
+  switch (tag) {
+    case "CAFE":
```

ESLint

[error] Expected indentation of 2 spaces but found 4.

(indent)